Stack/Stack/Stack.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <stdbool.h>
+
+// Structure for implementing a stack consisting of a value and a pointer to the next element
+typedef struct Stack
+{
+    int value;
+    struct Stack* next;
+}Stack;
+
+// Function for test empty stack
```

Comment:
```suggestion
// Function for testing if stack is empty
```
Например. Иначе как-то не по-английски.